lib/workers/pr/changelog/index.spec.ts

```diff
@@ -88,8 +88,17 @@ describe('workers/pr/changelog', () => {
       ).toBeNull();
       expect(httpMock.getTrace()).toHaveLength(0);
     });
-    it('works without Github', async () => {
-      httpMock.scope(githubApiHost);
+    it.skip('works without Github', async () => {
+      // httpMock
+      //   .scope(githubApiHost)
+      //   .get('/repos/chalk/chalk/tags?per_page=100')
+      //   .reply(200, [])
+      //   .get('/repos/chalk/chalk/releases?per_page=100')
+      //   .times(4)
+      //   .reply(200, [])
+      //   .get('/repos/chalk/chalk/contents/')
+      //   .times(4)
+      //   .reply(200, '');
```

Comment:
@zharinov Any idea why this test now fails? I know i changed the httpMock a litle bit, because it will throw otherwise. 🤔

Reply:
Hm, looks like you fixed `httpMock.ts` and this test now is failing correctly and should be fixed somehow

Reply:
@zharinov fixed